tests/unit/test_sentenceHelper.py

```diff
@@ -0,0 +1,39 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Rob Meredith
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+import documentNavigation.sentenceHelper as sentenceHelper
+
+
+class TestSentenceHelper(unittest.TestCase):
+	def test_findEndOfSentence(self):
+		westernSentences = (
+			"Test sentence number one. "
```

Comment:
Please add a test for missing space between sentences:
```
"Test sentence number one.Sentance number two."
```
For other whitespace between sentences (`\t`, `\n`, `\r`, `\n\r`).
Even if these are considered invalid, using the tests to document this behavior is helpful.